app/controllers/tournaments/games_controller.rb

```diff
@@ -7,6 +7,13 @@ class Tournaments::GamesController < ApplicationController
   def index
   end
 
+  def confirm
```

Comment:
This code is already in this controller in #update. Is the intention not to send notifications in this case?

Reply:
😕 app/controllers/games_controller.rb has an #update action but not the tournament one